### PR TITLE
bettermaps

### LINKED ISSUE #436: nick fixes
-[ ] put towers on position tab
-[ ] fix how ugly the heatmap coloras are
-[x] add tooltips to details kill deaths to explain the / notation
-[ ] confirm that teamfights death icon works

### PR DESCRIPTION
**Pull request by nickhh**
resolves #436

### CODE REVIEW

**Comment by howardc93** [comment]
a little blotchy IMO but signoff

**Comment by howardc93** [comment]
@albertcui opinion?

File: views/match/positions_map.jade [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
+  .col-md-5
+    h2 Positions
+    .table-responsive
+      table.table.table-hover
+
+        thead
+          tr
+            th Hero
+            th: abbr(title="position in the early game (sampled every second)") Lane
+            th: abbr(title="observer wards placed during the game") Observer
```
Comment (howardc93) [inline views/match/positions_map.jade]: capitalization?

Comment (howardc93) [inline views/match/positions_map.jade]: may want to note that it's "number of"

Comment (nickhh) [inline views/match/positions_map.jade]: the previous tool-tip didn't even work, and now you're complaining about capitalization? :P sigh, what a world...

**Comment by nickhh** [comment]
This says "nickhh wants to merge 4 commits into nocaptcha from bettermaps" which is wrong :P

**Pull request closed without merging**